include/grpcpp/impl/codegen/server_interface.h

```diff
@@ -219,7 +224,7 @@ class ServerInterface : public internal::CallHook {
     ServerCompletionQueue* const notification_cq_;
     void* const tag_;
     Message* const request_;
-    grpc_byte_buffer* payload_;
+    ByteBuffer payload_;
```

Comment:
This again is not an API change, IMO. Although it is changing a protected member which could technically be viewed by a subclass, it was actually a style violation to have protected members at all. As a result, they're not considered API. So this is fine too.